### ISSUE TITLE
[1.12.1] Draconic Reactor explodes upon restarting the server even though it was fully stable before

### ISSUE THREAD

**Issue by Tremmert9** [brandon3055/Draconic-Evolution#908]
## Description
The Draconic Reactor keeps exploding whenever the server restarts, even though it was perfectly stable prior to the server shutting down.
![image](https://user-images.githubusercontent.com/20724786/30514579-266c5a30-9b18-11e7-9e8c-993aaf9ef8e6.png)

( If the above picture doesn't load, the link to it is here - https://puu.sh/xBfOV/0d78cbf273.jpg )

The above picture is the latest picture I have of our reactor being stable. I have managed to kick it to 1,6M RF/t, fully stable (similar stats to the picture above) before the server had to be restarted. Upon starting the server, we are either greeted with the reactor status of "Explosion Imminent" on the reactor or just a big boom when we connect to the server.

Is there a way to fix this issue? We had a fully stable reactor that only explodes because of the server restarting, and our reactor is placed under our base for convenience sake, as we know how to properly manage the reactor, but wouldn't ever think that the reactor would explode due to a server restart.
Right now we have to keep the server offline until we find a fix, and if the fix even includes just deleting the reactor from the world prior to starting the server, we will take that. But for long-term, we would like to know if this issue happened before and if there is a fix for it.

## Base information
* Minecraft version: 1.12.1
* Minecraft Forge version: 14.22.0.2467
* Mod Pack: All The Mods 3, v3.1

## Crash report
```
N/A
```

**Comment by Express93** [comment]
Working on this with Tremmert. I used MCEdit to manipulate the block data and set "reactorState" to 0, this allowed the core to cool down then we broke the block and replaced, upon configuring it and powering it back up it just became instantly unstable again.

**Comment by brandon3055** [comment]
Is the reactor chunk loaded? (With a sane chunk loader mod like Chicken Chunks) 
The only possible cause i can think of is chunk loading order when the server starts/you join the world. If the chunk with the reactor in it is loading a second or 2 before the chunk with the energy storage powering the shield that may be long enough for the shield to run out of power at that high power level.

**Comment by Tremmert9** [comment]
The reactor is chunk loaded using an IC2 Chunk Loader - unfortunately ATM 3 modpack doesn't leave us much of a choice, so we have to use that one, as we've got no alternatives.

We've found a temporary solution, and that is to go in the server config files and manually turn off the reactor, then hop into the server - the reactor is then turned off and poses no threat. Though that kind of a shutdown system is not something we'd prefer, as we need to wait a good couple of minutes for the reactor to be stable enough with shield and low enough in temperature to shut it off without an issue. That's how we've been doing it so far and that has worked.

However, there was a hiccup that we encountered only one time after fiddling with server files to shut the reactor off manually - we mined the core out and replaced it with a new one, had the same exact setup and we turned the reactor on. The reactor did not receive absolutely any kind of input power whatsoever, which caused it to drop from 50% to 0% shields and explode (even though the reactor managed to raise shields and warm up perfectly fine prior to starting it up).
After a ton of troubleshooting, it seems that the input flux gate has simply not worked - destroying it and replacing it with a new one with the same amount of RF/t fixed the issue. Why that flux gate simply stopped working, I still do not know to this day, but perhaps maybe it has to do something with the reactor exploding in the first place on server restart? Or maybe it stopped working after we turned the reactor off manually in server files?

Apologies for the wall of text - just trying to tell my side of the story to help you understand where the problem might lay, and to even fix a possible bug that might exist.
